### ISSUE TITLE
CSS transform scale breaks column resize

### ISSUE THREAD

**Issue by shepardtone** [handsontable/handsontable#3946]
### Description
When the parent element of a table has the CSS property `transform: scale` applied, column resizing no longer works.
For context, I am using the table in a design application that has a zoom feature, where we apply scale transforms to editable content.

### Steps to reproduce

1. Create a container `<div>` with CSS style "transform: scale(1)" (although it breaks at any scale)
2. Create a `<div>` inside the container
3. Create a Handsontable instance on the inner `<div>`
4. Column resizing will not work, unless the scale transform is removed from the parent `<div>`.

### Demo
See demo here: http://jsfiddle.net/e4g781r7/2/

### Your environment
Tested using:
* Handsontable version 0.29.1
* Chrome 54
* OSX 10.11.6

**Comment by AMBudnik** [comment]
Thanks for sharing @shepardtone 
You're right the moving bar is lost after setting `transform` attribute.

**Comment by AMBudnik** [comment]
strongly related https://github.com/handsontable/handsontable/issues/3946

**Comment by snarkitect** [comment]
I'm blocked by this same bug.

**Comment by AMBudnik** [comment]
Replicable in v0.34.3

**Comment by AMBudnik** [comment]
The issue is replicable in v `2.0.0.` and doesn't seem to work fine in any of the previous versions.

**Comment by toroc** [comment]
We are also blocked by this and are using 3.0.0.

**Comment by wojciechczerniak** [comment]
Housekeeping. We have few issues related to this one. I'm closing this one as a duplicate of #5050. Please track #5050 for further development.

**Comment by AMBudnik** [comment]
Hey @shepardtone @snarkitect @toroc we've just published Handsontable v `7.0.0` were he fix a referred issue https://github.com/handsontable/handsontable/issues/5050
Please update and let me know if it works for you.
